tests/epyccel/modules/generic_functions.py

```diff
@@ -108,7 +83,7 @@ def tst_gen_5():
 def tst_gen_6():
     x = gen_6(5.5, 5.5)
     y = gen_6(5, 5)
-    z = gen_6(1j, 1j)
+    z = gen_6(complex(1, 2), complex(1, 2))
```

Comment:
Is a more Pythonic syntax also accepted? E.g.
```python
z = gen_6(1+2j, 1+2j)
```

Reply:
yes, It completely fine I'm gonna add it in some tests